packages/react-components/react-toast/src/state/vanilla/getPositionStyles.ts

```diff
@@ -0,0 +1,44 @@
+import { ToastPosition } from '../types';
+
+interface PositionStyles {
+  position: 'fixed';
+  top?: number;
+  left?: number;
+  right?: number;
+  bottom?: number;
+}
+
+export const getPositionStyles = (position: ToastPosition) => {
+  const positionStyles: PositionStyles = {
+    position: 'fixed',
+  };
+
+  switch (position) {
+    case 'top-left':
+      Object.assign(positionStyles, {
```

Comment:
Curiosity: why `Object.assign()` over a regular spread?

Reply:
I've started using Object.assign recently when I don't need to create new objects. Just a habit, I can use spread if you prefer